core/models.py

```diff
@@ -165,3 +168,30 @@ def image_url(self: "Image") -> str:
             return self.image.url
         except ValueError:
             return ""
+
+
+class CommunityMember(models.Model):
+    ADMIN: ClassVar[str] = "admin"
+    MODERATOR: ClassVar[str] = "moderator"
+    MEMBER: ClassVar[str] = "member"

```

Comment:
```suggestion
    MEMBER: ClassVar[str] = "100_member"
    MODERATOR: ClassVar[str] = "200_moderator"
    ADMIN: ClassVar[str] = "300_admin"
```
this way we can use number comparison if user has certain privileges. 
```
a = "100_x"
int(a.split("_")[0]) > 1
```
Also we are keeping member smallest as it has least privileges.